packages/cli/src/commands/pull.ts

```diff
@@ -30,7 +37,8 @@ const help = () => {
     'DIR'
   )}    Path to the global ${'`.vercel`'} directory
     -d, --debug                    Debug mode [off]
-    --env [filename]               The file to write Development Environment Variables to [.env]
+    --env-file [filename]          The file to write Development Environment Variables to [.env]
+    --environment ${getEnvTargetPlaceholder()} Deployment environment [development]
```

Comment:
```suggestion
    --environment [environment] Deployment environment [development]
```

This one can be changed to just a name. Then lets add examples below with the target placeholder expanded like in `vc env --help`